plugins/ftdidmx/FtdiDmxPlugin.cpp

```diff
@@ -139,8 +138,9 @@ bool FtdiDmxPlugin::SetDefaultPreferences() {
 int unsigned FtdiDmxPlugin::GetFrequency() {
   unsigned int frequency;
 
-  if (!StringToInt(m_preferences->GetValue(K_FREQUENCY), &frequency))
-    StringToInt(DEFAULT_FREQUENCY, &frequency);
+  if (!StringToInt(m_preferences->GetValue(K_FREQUENCY), &frequency)) {
```

Comment:
I'd invert this, I think it's easier to read :+1: 

if (StringToInt(m_preferences->GetValue(K_FREQUENCY), &frequency)) {
  return frequency;
}
return DEFAULT_FREQUENCY;